--- a/browser/locales/jar.mn
+++ b/browser/locales/jar.mn
@@ -17,6 +17,7 @@
   preview/cookieBannerPreferences.ftl              (../components/preferences/cookieBannerPreferences.ftl)
   locales-preview/migrationWizard.ftl              (../locales-preview/migrationWizard.ftl)
   browser                                          (%browser/**/*.ftl)
+  librewolf                                        (%librewolf/**/*.ftl)
 
 @AB_CD@.jar:
 % locale browser @AB_CD@ %locale/browser/
